pre-docs/inventory_version2.md

```diff
@@ -39,6 +39,7 @@ string to a target defined elsewhere in the inventory. If no target has a name
 or alias matching the string bolt will create a new target with the string as
 it's uri.
 
+**Note:** In addition to inventory file changes, inventory functions such as `get_targets` cannot be called from an [apply block](applying_manifest_blocks.md) using inventory V2
```

Comment:
I think make this a whole new section. "Updating plans" With an example of extracting information at the plan level into a variable. 

```
plan setup_lb (
   TargetSpec $pool_members,
   TargetSpec $lb
) {
   apply_prep([$pool_members, $lb])

   $member_hosts = get_targets($pool_members).map |$targ| { target.host }
   apply($lb) {
       class { 'profile::lb':
          members => $member_hosts
       }
   }
}
```

Reply:
you could also extract an ip address from the facts of the pool members.